product-backend/scripts/create_db.sql

```diff
@@ -7,3 +7,13 @@ create table JobRoles (
     job_role_id INT primary key AUTO_INCREMENT,
     job_role_title varchar(100)
 );
+
+
+--US024 registration system
+create table `User` (
```

Comment:
Fix the previous table too
```suggestion
CREATE TABLE IF NOT EXISTS `User` (
```